Jenkinsfile

```diff
@@ -56,13 +56,17 @@ stage('Record builds and sessions') {
           // TODO Launchable not yet supported on LTS line
           return
         }
+        infra.withArtifactCachingProxy {
+          sh "DISPLAY=:0 ./run.sh firefox ${jenkinsVersion} -Dmaven.repo.local=${WORKSPACE_TMP}/m2repo -B clean process-test-resources"
+        }
```

Comment:
Using this script to fetch the WAR file, the same way we would a few lines down below. The `DISPLAY=:0` and Firefox arguments aren't needed for downloading the WAR file, but they are needed for running this script.